app/staking/staking.module.scss

```diff
@@ -114,7 +117,7 @@
 }
 
 .tableTitleContainer {
-  width: 55%;
+  width: 72%;
```

Comment:
please avoid percentages that are not 100%. I've noticed that even spacing between tables and sidebar is 2% margin avoid that, and stick to fixed px/rem values

Reply:
Will remove the 2% margin. But to make the Tables width consistent on large screens too, I was using percentages like 70 etc. will try to replace these with rem/px values

Reply:
Added rem instead of percentages for margins